topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -652,7 +652,7 @@ Another option is to estimate these parameters with a tool called **Infer Experi
 > ### {% icon hands_on %} Hands-on: Determining the library strandness
 >
 > 1. **Convert GTF to BED12** {% icon tool %} to convert the GTF file to BED:
```

Comment:
This is the second time this step is performed, can't we just use the previous output?

Reply:
I removed this step and pointed the user towards the previously created output.